modules/boost.units/1.83.0.bcr.1/MODULE.bazel

```diff
@@ -0,0 +1,20 @@
+module(
+    name = "boost.units",
+    version = "1.83.0.bcr.1",
```

Comment:
Should be `1.83.0` -  would be `1.83.0.bcr.1` if a bug fix to the bazelization is done after version `1.83.0`  has been published

Reply:
Well, in this specific case, all the other boost modules have this version. Here's a snippet from how I and (I suspect) most people will use these boost libraries.

```
BOOST_VERSION = "1.83.0.bcr.1"

bazel_dep(name = "boost.config", version = BOOST_VERSION)
bazel_dep(name = "boost.core", version = BOOST_VERSION)
bazel_dep(name = "boost.filesystem", version = BOOST_VERSION)
bazel_dep(name = "boost.format", version = BOOST_VERSION)
bazel_dep(name = "boost.geometry", version = BOOST_VERSION)
bazel_dep(name = "boost.graph", version = BOOST_VERSION)
bazel_dep(name = "boost.iterator", version = BOOST_VERSION)
bazel_dep(name = "boost.lexical_cast", version = BOOST_VERSION)
bazel_dep(name = "boost.optional", version = BOOST_VERSION)
bazel_dep(name = "boost.polygon", version = BOOST_VERSION)
bazel_dep(name = "boost.program_options", version = BOOST_VERSION)
bazel_dep(name = "boost.property_map", version = BOOST_VERSION)
bazel_dep(name = "boost.serialization", version = BOOST_VERSION)
bazel_dep(name = "boost.type_traits", version = BOOST_VERSION)
bazel_dep(name = "boost.units", version = BOOST_VERSION)
bazel_dep(name = "boost.variant", version = BOOST_VERSION)
```

If we use `1.83.0` here instead of `1.83.0.bcr.1`, it would be something like this in my MODULE.bazel file:

```
BOOST_VERSION = "1.83.0.bcr.1"

bazel_dep(name = "boost.config", version = BOOST_VERSION)
bazel_dep(name = "boost.core", version = BOOST_VERSION)
bazel_dep(name = "boost.filesystem", version = BOOST_VERSION)
bazel_dep(name = "boost.format", version = BOOST_VERSION)
bazel_dep(name = "boost.geometry", version = BOOST_VERSION)
bazel_dep(name = "boost.graph", version = BOOST_VERSION)
bazel_dep(name = "boost.iterator", version = BOOST_VERSION)
bazel_dep(name = "boost.lexical_cast", version = BOOST_VERSION)
bazel_dep(name = "boost.optional", version = BOOST_VERSION)
bazel_dep(name = "boost.polygon", version = BOOST_VERSION)
bazel_dep(name = "boost.program_options", version = BOOST_VERSION)
bazel_dep(name = "boost.property_map", version = BOOST_VERSION)
bazel_dep(name = "boost.serialization", version = BOOST_VERSION)
bazel_dep(name = "boost.type_traits", version = BOOST_VERSION)
bazel_dep(name = "boost.units", version = "1.83.0")
bazel_dep(name = "boost.variant", version = BOOST_VERSION)
```

It'd be slightly weird but not really. Just wanted to confirm. Happy to make the change.

Reply:
Although thinking that through a bit deeper -- if any of these boost packages have a minor bug that needs to be fixed, that would mean all of them have to have the version bumped, which probably isn't desirable.